css/main.css

```diff
@@ -136,12 +184,61 @@ header .menu img {
   margin-left: 1rem;
 }
 
+@media (min-width: 768px) {
+  .hero__container {
+    background-image: url(../images/illustration-header-desktop_cut.svg);
+  }
```

Comment:
- [ ] Kindly fix the issue with this section. When the screen is smaller it becomes like this:

![image](https://user-images.githubusercontent.com/50721479/186996612-87bdc3f4-223d-4e87-b461-8cbb6df52064.png)